types/tedious-connection-pool/package.json

```diff
@@ -7,7 +7,7 @@
     ],
     "dependencies": {
         "@types/node": "*",
-        "@types/tedious": "*"
+        "tedious": "*"
```

Comment:
This might need to be pinned back; look how old the package version is upstream: https://github.com/tediousjs/tedious-connection-pool/blob/2b8ca4b6c18b080b1449d4d866540d7cc53ed379/package.json#L34